projects/bp-gallery/src/shared/locales/de.json

```diff
@@ -382,5 +389,43 @@
     "locations": "Orte",
     "more-info": "Wissen Sie mehr über diese Bilder?",
     "our-categories": "Unsere Kategorien"
+  },
+  "tag-panel": {
+    "name-of-sub-keyword": "Name des neuen Unterschlagworts für {{ parent }}?",
+    "name-of-sub-location": "Name des neuen Unterorts für {{ parent }}?",
+    "name-of-keyword": "Name des neuen Schlagworts?",
+    "name-of-location": "Name des neuen Orts?",
+    "add-sub-keyword": "Unterschlagwort für {{ parent }} hinzufügen",
+    "add-sub-location": "Unterort für {{ parent }} hinzufügen",
+    "add-location": "Ort hinzufügen",
+    "add-keyword": "Schlagwort hinzufügen",
+    "new-location-name": "Neuer Name des Ortes {{ tag_name }}?",
+    "new-keyword-name": "Neuer Name des Schalgworts {{ tag_name }}?",
+    "detach-location": "Diesen Ort aus der Hierarchie lösen?",
+    "detach-keyword": "Dieses Schalgwort aus der Hierarchie lösen?",
+    "should-delete-location": "Soll folgender Ort und alle seine Unterorte gelöscht werden?",
+    "should-delete-keyword": "Soll folgendes Schlagwort und alle seine Unterschlagworte gelöscht werden?",
+    "just-delete-single-location": "Nur diesen Ort löschen",
+    "just-delete-single-keyword": "Nur dieses Schalgwort löschen",
+    "relocate-location": "Welchem Ort wollen Sie folgenden Ort unterordnen?",
+    "relocate-keyword": "Welchem Schlagwort wollen Sie folgendes Schlagwort unterordnen?",
+    "location": "Ort",
+    "keyword": "Schlagwort",
+    "multiple-paths": "Mehrere Pfade",
+    "select-position": "An welcher Position möchten Sie {{ name }} hinzufügen?",
+    "location-already-exists": "Es existiert bereits ein Ort mit dem Namen {{ name }} an der Stelle, wo sie en Ort einfügen wollen. Diese Operation wird deswegen nicht ausgeführt.",
+    "delete-elsewhere": "Der Ort {{ name }} besitzt eine Kopie an anderer Stelle. Soll diese auch gelöscht werden?",
+    "only-delete-here": "Nur hier löschen.",
+    "delete-everywhere": "Überall löschen.",
+    "loop-error": "Diese Aktion würde eine Kreisschluss der Orte zur Folge haben und wird deswegen nicht ausgeführt.",
+    "location-pictures": "{{ amount }} Bild/er in diesem Ort"
```

Comment:
you can use [this](https://www.i18next.com/translation-function/plurals) to get rid of the `/` (you also need to change `amount` to `count`)